Bottom: 822a854059a13358319b97d9dab6fed957084082
Top:    71d5ecbf50720877e5e802cbc00c57e840479039
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 22:10:05 +0100

Refresh of place-drawing-of-crosshair-att

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 24f51ab..f733e11 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -85,7 +85,6 @@ static void XORDrawMoveOrCopyObject (void);
 static void XORDrawAttachedLine (LocationType, LocationType, LocationType,
 				 LocationType, BDimension);
 static void XORDrawAttachedArc (BDimension);
-static void DrawAttached (bool);
 
 /* ---------------------------------------------------------------------------
  * creates a tmp polygon with coordinates converted to screen system
@@ -566,7 +565,7 @@ XORDrawMoveOrCopyObject (void)
 /* ---------------------------------------------------------------------------
  * draws additional stuff that follows the crosshair
  */
-static void
+void
 DrawAttached (void)
 {
   BDimension s;
diff --git a/src/crosshair.h b/src/crosshair.h
index 4573e98..2b9eed3 100644
--- a/src/crosshair.h
+++ b/src/crosshair.h
@@ -46,7 +46,7 @@
 #define	STATE_SECOND	1
 #define	STATE_THIRD		2
 
-
+void DrawAttached (void);
 void CrosshairOn (void);
 void CrosshairOff (void);
 void HideCrosshair (void);
